pandas/core/indexes/multi.py

```diff
@@ -1847,18 +1847,23 @@ def cats(label):
 
     def sortlevel(self, level=0, ascending=True, sort_remaining=True):
         """
-        Sort MultiIndex at the requested level. The result will respect the
-        original ordering of the associated factor at that level.
+        Sort MultiIndex at the requested level.
+
+        Sort MultiIndex at the requested level and optionally the
```

Comment:
I'd avoid repeating the short summary, I don't think the extended summary is in any case presented without the short first. You can start with the "Optionally..."